packages/commands/src/index.ts

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+import { program } from '@commander-js/extra-typings'
+
+import chalk from 'chalk'
+import type { ResponseError } from '@audius/sdk'
+import { managerCommand } from './manager/account-managers.js'
+import { authHeadersCommand } from './misc/auth-headers.js'
+import {
+  claimRewardCommand,
+  rewardSpecifierCommand
+} from './misc/claim-reward.js'
+import { createUserBankCommand } from './misc/create-user-bank.js'
+import { mintCommand } from './misc/mint.js'
+import { tipReactionCommand } from './misc/tip-reaction.js'
+import { withdrawTokensCommand } from './misc/withdraw-tokens.js'
+import { playlistCommand } from './playlist/index.js'
+import { trackCommand } from './track/index.js'
+import { userCommand } from './user/index.js'
+import { albumCommand } from './album/index.js'
+
+async function main() {
+  program.name('audius-cmd')
+  program.description(
+    'CLI interface for interacting with a local Audius network.'
+  )
+  program.addCommand(userCommand)
+  program.addCommand(trackCommand)
+  program.addCommand(playlistCommand)
+  program.addCommand(albumCommand)
+  program.addCommand(managerCommand)
+  program.addCommand(authHeadersCommand)
+  program.addCommand(claimRewardCommand)
+  program.addCommand(rewardSpecifierCommand)
+  program.addCommand(createUserBankCommand)
+  program.addCommand(mintCommand)
+  program.addCommand(tipReactionCommand)
+  program.addCommand(withdrawTokensCommand)
+
+  try {
+    await program.parseAsync(process.argv)
+  } catch (err) {
+    if ('response' in (err as Error)) {
+      const resErr = err as ResponseError
+      console.log(
```

Comment:
This is interesting. Were we already doing this? I don't feel like request IDs are super useful in local stack

Reply:
They're somewhat useful! I used them for rewards claims. Probably not going to be as useful most the time, can revisit